config/activator.yaml

```diff
@@ -53,14 +50,38 @@ spec:
               fieldRef:
                 fieldPath: metadata.name
         volumeMounts:
+        - name: localpods
+          mountPath: /etc/localpods
         - name: config-logging
           mountPath: /etc/config-logging
         - name: config-observability
           mountPath: /etc/config-observability
       volumes:
-        - name: config-logging
-          configMap:
-            name: config-logging
-        - name: config-observability
-          configMap:
-            name: config-observability
+      - name: localpods
+        hostPath:
+          path: /etc/kubernetes/manifests
```

Comment:
Is this safe, what I mean is would this be considered a security risk? Don't we have the k8s backplane and all that jazz in that directory?

Reply:
Also another question most notably due to my ignorance here but do we need a localmount PV for this?

Reply:
I am not sure we need a pv since we dont really need to do some magic to persist the data - the hypervisor does it for us. Theres definitely some issues that need to be worked out here though:

- This path isn't something we can depend on (its operator specific / set via kubelet config).
- As you point out this allows us to create (and read) arbitrary static pod definitions on the cluster. Potential security implications here.